lib/python/qmk/cli/lint.py

```diff
@@ -46,6 +48,11 @@ def lint(cli):
         ok = False
         cli.log.error('Missing %s', readme_path)
 
+    # Check keyboard name is valid
+    if not re.fullmatch(r"[a-z0-9][a-z0-9_]*", cli.config.lint.keyboard):
```

Comment:
This regex doesn't match keyboards that use subdirectories... like... your own: :laughing: 

```
$ qmk lint -kb zvecr/zv48
ERROR Invalid keyboard name
ERROR Lint check failed!
```

```suggestion
    if not re.fullmatch(r"[a-z0-9][a-z0-9_/]*", cli.config.lint.keyboard):
```

Reply:
Good catch!